e2e-tests/tfm/cypress/e2e/pages/utilisation-reports/utilisationReportPage.js

```diff
@@ -12,6 +12,12 @@ const utilisationReportPage = {
     generateKeyingDataButton: () => cy.get('[data-cy="generate-keying-data-button"]'),
     addAPaymentButton: () => cy.get('[data-cy="add-a-payment-button"]'),
     premiumPaymentsTable: {
+      header: {
+        totalReportedPayments: () => cy.get('th[data-cy="premium-payments-table--total-reported-payments"] button'),
+        totalPaymentsReceived: () => cy.get('th[data-cy="premium-payments-table--total-payments-received"] button'),
```

Comment:
Is it possible to add a `data-cy`to the button, or is this not controlled by us?

Reply:
The button is inserted by the MOJ frontend script I believe so I don't think we can pass any attributes down.

I tried this without the `button` and the tests don't wait for the script to manipulate the DOM (to add the button) - so the sorting wouldn't be tested correctly without selecting the button element itself